db/seeds.rb

```
@@ -5,3 +5,25 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Image.create(url: 'https://i.imgur.com/8c8mTIe.gif?noredirect')
```

So two things:

1. If creating an image fails we should raise an exception since we expect this to _always_ work
2. There is a way to create all 20 images with a single method call that involves passing an array to the method - I'll let you find the exact api for it

this is also a good chance to practice getting familiar with your editor - both VSCode and RubyMine have support for adding multiple cursors 🙂

---

👍